src/DynamoCoreWpf/Views/SplashScreen/SplashScreen.xaml.cs

```diff
@@ -444,6 +443,12 @@ private void CloseWindow()
                 Application.Current.Shutdown();
                 Analytics.TrackEvent(Actions.Close, Categories.SplashScreenOperations);
             }
+            else if (this is SplashScreen)
+            {
+                this.Close();
+                viewModel.Model.ShutDown(false);
+                DynamoModel.RequestUpdateLoadBarStatus -= DynamoModel_RequestUpdateLoadBarStatus;
```

Comment:
Exellent! You found the magic bullet. Would you add some comments to this section so people understand the reasoning?